### DISCUSSION TITLE
Setup does not run properly

### DISCUSSION THREAD

**Discussion by megatoeter** [MiczFlor/RPi-Jukebox-RFID#2018] [Q&A] [answered]
```
  CONFIGURE WIFI
 
 Requires SSID, WiFi password and the static IP you want
 to assign to your Phoniebox.
 (Note: can be done manually later, if you are unsure.)

 Do you want to configure your WiFi? [Y/n] n

 You want to configure WiFi later.
 ###  **./buster-install-default.sh: line 159: /home/pi/PhonieboxInstall.conf: No such file or directory
 ### ./buster-install-default.sh: line 161: /home/pi/PhonieboxInstall.conf: No such file or directory**
 Hit ENTER to proceed to the next step.^C
```

I get this error everytime I run this command:
`cd; rm buster-install-*; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/master/scripts/installscripts/buster-install-default.sh; chmod +x buster-install-default.sh; ./buster-install-default.sh`

I am using a Pi4 Model B and the latest Raspberry Pi OS (tried standard and lite) 32Bit.

I would appreciate any help :D

**Comment by bjentsch** [comment, marked as answer]
In which directory are you running the command? You should be in `~` and running `pwd` should output `/home/pi`.
